codes/quantum/qubits/subsystem/bacon_shor.yml

```diff
@@ -43,6 +43,7 @@ features:
 
 realizations:
   - 'Trapped-ion qubits: state preparation, logical measurement, and stabilizer measurement for nine-qubit Bacon-Shor code demonstrated on a 13-qubit device by M. Cetina and C. Monroe groups \cite{arxiv:2009.11482}.'
+  - 'Photons: quantum teleportation of information implemented on maximally entangled pair of physical and logical qubits with fidelity rate of up to 78.6\% \cite{arXiv:2009.06242}.'
```

Comment:
Nice. Please list how many physical qubits.

Reply:
It was one of each, I listed it in the recent commit.